benchto-driver/src/test/resources/unit-benchmarks/simple-benchmark.yaml

```diff
@@ -1,3 +1,4 @@
+foo: 42
```

Comment:
please do not use names like `foo` and `bar`. IMO the best name would be 'overridable` with value `not yet overridden`, and then you could change `bar` to `overriden`